src/app/components/SubjectHeading/Search/SubjectHeadingSearch.jsx

```diff
@@ -35,6 +35,8 @@ class SubjectHeadingSearch extends React.Component {
       if (this.state.userInput) {
         return axios(`${appConfig.baseUrl}/api/subjectHeadings/autosuggest?query=${this.state.userInput}`)
           .then((res) => {
+            const numberOfResults = res.data.autosuggest.length;
+            document.getElementById('subject-heading-content-primary').style['min-height'] = `${(numberOfResults > 12 ? numberOfResults : 12) * 50}px`
```

Comment:
Is `document.getElementById` discouraged in React or nah? Is it possible `document.getElementById('subject-heading-content-primary')` is going to match 0 nodes and this will error?